### ISSUE TITLE
The aggregate dash shows missed proposal for Mostly Harmless cluster in the last 7 days

### ISSUE THREAD

**Issue by boulder225** [ObolNetwork/charon#2189]
The aggregate dash shows missed proposal for Mostly Harmless cluster in the last 7 days, however looking at the 5 validators of this cluster, I can't find any missed proposal in beaconcha.in, is this a bug or am I missing something?

![Image](https://user-images.githubusercontent.com/68026093/237990500-88c6afab-1d88-417b-bbf4-9cee118e056c.png)

Would be nice to also add attestations success and missed in this table so we can share it in the Lido channel

**Comment by corverroos** [comment]
I changed the dash to only show failed proposals "for the given time window" only, not "since restart of the nodes".

**Comment by gsora** [comment]
While the cluster did experience a missed block proposal on the 10th of April, the Grafana board (see picture) was erroneously showing the maximum amount of failed proposal _ever_:  it wasn't respecting the Grafana time window

The 1 that we've been seeing was the result of a different issue: even if one of the cluster component didn't successfully broadcast the block proposal, it would get reported as a completely failed proposal.

![image](https://github.com/ObolNetwork/charon/assets/145909/fa88efae-019d-4647-96be-ee0f5f0c7b54)

Tn this image you can see something happened on May 2: two of the 8 nodes were not able to broadcast the block proposal, hence marking it as failed for the entire cluster.

The broadcast failed because of reasons unrelated to charon, but due to beacon node behavior.

Bottom line: always check is to see if there was "submit_X" beacon node API errors during that time window, and also metrics are relative to the node being observed.

One must always think at the cluster level, but also checking individual cluster components to have a real vision on the situation.
